scripts/github_api

```diff
@@ -0,0 +1,106 @@
+#!/bin/python
```

Comment:
Could you make the shebang consistent in all executable scripts (eg host_os.py) in a separate patch?